shared/elements/gaia_switch/style.css

```diff
@@ -47,7 +47,7 @@ gaia-switch {
   content: '';
   position: absolute;
   right: 0;
-  top: 50%;
+  top: 2.5rem;
```

Comment:
So this will change the switch to always be near the top of the label. We can follow what we did previously, but it does seem a bit weird/inconsistent to me. Happy to implement how you think is best though, previously we did: 

label only -> switch is at vertical center
label + details -> switch is aligned at top

The problem I see with that is what happens when you have a label that wraps multiple lines as well as a details object. In that case the switch would be aligned near the top, so why not always align at the top? Maybe that's not really a common use-case though.